Tools/OSD/osd_screen.cs

```diff
@@ -690,8 +713,17 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
         private void pictureBox1_MouseDown (object sender, MouseEventArgs e) {
 			osd.BeginInvoke((MethodInvoker)delegate {
 				osd.currentlyselected = getMouseOverItem(e.X, e.Y);
-                adjustGroupbox();
-			});
+                adjustGroupbox();

```

Comment:
Is working with the control of of another thread should not be done through the Invoke?